test/dynamo/test_error_messages.py

```diff
@@ -1072,6 +1078,146 @@ def hn(x):
     hn(x + 1)
   File "test_error_messages.py", line N, in hn
     torch._dynamo.graph_break()  # 1
+Most recent bytecode instructions traced (max 20):
+""",
+        )
+
+    @torch._dynamo.config.patch(verbose=True)
+    @make_logging_test(graph_breaks=True)
+    def test_variable_tracker_bytecode_to_graph_break_fullgraph(self, records):
+        # def test_bytecode_graphbreak_fullgraph(self, x):
+        def fn(x):
+            y = x + 1
+            z = x + y
+            torch._dynamo.graph_break()
+            return z
+
+        # assertRaises suppresses the traceback, so manually catch
+        e = None
+        try:
+            torch.compile(fn, backend="eager", fullgraph=True)(torch.ones(3))
+        except Exception as exn:
+            e = exn
+
+        self.assertIsNotNone(e)
+        msg = "".join(traceback.format_exception(type(e), e, e.__traceback__))
+        self.assertExpectedInline(
+            munge_exc(msg, suppress_suffix=True, skip=0),
+            """Traceback (most recent call last):
+  File "test_error_messages.py", line N, in test_variable_tracker_bytecode_to_graph_break_fullgraph
+    torch.compile(fn, backend="eager", fullgraph=True)(torch.ones(3))
+torch._dynamo.exc.Unsupported: Call to `torch._dynamo.graph_break()`
+  Explanation: User-inserted graph break. Message: None
+  Hint: Remove the `torch._dynamo.graph_break()` call.
+
+  Developer debug context: Called `torch._dynamo.graph_break()` with args `[]`, kwargs `{}`
+
+ For more details about this graph break, please visit: https://meta-pytorch.github.io/compile-graph-break-site/gb/gb0025.html
+
+from user code:
+   File "test_error_messages.py", line N, in fn
+    torch._dynamo.graph_break()
```

Comment:
are you checking for bytecode here? or this will be a followup? if it's a followup, can you add this test there instead?